package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hapi-postgres-connection",
-  "version": "6.1.0",
+  "version": "6.1.1",
```

Comment:
@naazy are you upgrading the version of `node-pg` in `package.json > dependencies` ?